packages/opentelemetry-node/lib/detectors.js

```diff
@@ -97,6 +97,14 @@ function resolveDetectors(detectors) {
         return [];
     }
 
+    // NOTE: Kibana is doing a breakdown by service instance when plotting the metrics dashboard. So it needs the instance ID.
+    // Dashboard was added in https://github.com/elastic/kibana/pull/215735
+    if (!detectorKeys.includes('serviceinstance')) {
+        log.info(
```

Comment:
Did you intentionally select INFO-level so it is below WARN and can be effectively ignored via `OTEL_LOG_LEVEL=warn`?

These feel a *bit* like they should be warnings. However, the first thing that will be requested by a user with a legitimate use case for this will be "how can I disable this warning".   We don't want to get into having a richer warnings system where particular warnings can be skipped with configuration -- certainly not now, and perhaps not ever (for example see Python's warning message controls: https://docs.python.org/3/library/warnings.html). So I think "info" level is fine.